src/android/orb.yml

```
@@ -218,4 +218,26 @@ commands:
             [[ -n "$CIRCLE_BRANCH" ]] && PARAMS+=(--branch-name=$CIRCLE_BRANCH)
             [[ -n "$CIRCLE_PULL_REQUEST" ]] && PARAMS+=(--pull-request-url=$CIRCLE_PULL_REQUEST)
             ./gradlew --stacktrace :<<parameters.module_name>>:publishToS3 "${PARAMS[@]}"
+  should-publish-library-artifacts:
+    description: Check if library artifacts should be published
+    steps:
+      - run:
+          name: Check if library artifacts should be published
+          command: |
+            if [[ -z "$CIRCLE_SHA1" ]]; then
+                echo "Commit hash should always be available. Since we rely on this assumption in later commands, this acts as a sanity check"
```

```suggestion
                echo "Commit hash should always be available. Since we rely on this assumption in later commands, this acts as a precondition check"
```

Rationale: it makes the message [more inclusive](https://english.stackexchange.com/questions/282282/near-universally-applicable-alternative-to-sanity-check).